### PR TITLE
OCPBUGS-35895: Enable fips in iso kernel options on fips systems

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by elfosardo** [comment]
/override ci/prow/okd-images
unrelated

File: scripts/copy-iso [hunk 1]
```diff
@@ -23,7 +23,10 @@ copy_if_needed() {
         if [ -n "${IP_OPTIONS:-}" ]; then
             echo "Adding kernel argument ${IP_OPTIONS}" >&2
             rm -f "${dest_file}"
-            coreos-installer iso kargs modify -a "${IP_OPTIONS}" -o "${dest_file}" "${source}"
+            if [[ -f /proc/sys/crypto/fips_enabled ]]; then
+                ENABLE_FIPS_IPA=$(cat /proc/sys/crypto/fips_enabled)
+            fi
+            coreos-installer iso kargs modify -a fips="${ENABLE_FIPS_IPA:-0}" "${IP_OPTIONS}" -o "${dest_file}" "${source}"
```
Comment (derekhiggins) [inline scripts/copy-iso:29]: the "-a" looks funny to me, can you pass it 2 args ?  
it might be right just double checking

**Comment by derekhiggins** [comment]
/hold
waiting for response to question

File: scripts/copy-iso (hunk 1 above)
Comment (elfosardo) [reply scripts/copy-iso:29]: thanks Derek, this is actually wrong, I think I messed up my local repo, going to submit the actual change later

File: scripts/copy-iso [hunk 2]
```diff
@@ -20,10 +20,20 @@ copy_if_needed() {
...
         echo "Extracting ISO file" >&2
+        KERNEL_ARGS=""
         if [ -n "${IP_OPTIONS:-}" ]; then
             echo "Adding kernel argument ${IP_OPTIONS}" >&2
+            KERNEL_ARGS+="${IP_OPTIONS} "
+        fi
+        if [ -f /proc/sys/crypto/fips_enabled ]; then
+            ENABLE_FIPS_IPA=$(cat /proc/sys/crypto/fips_enabled)
+            echo "Adding kernel argument fips=${ENABLE_FIPS_IPA:-0}" >&2
+            KERNEL_ARGS+="fips=${ENABLE_FIPS_IPA:-0} "
```
Comment (derekhiggins) [inline scripts/copy-iso:31]: We might need to only set this if enabled 
https://issues.redhat.com//browse/OCPBUGS-39536

**Comment by derekhiggins** [comment]
/hold
see comment inline

File: scripts/copy-iso (hunk 2 above)
Comment (elfosardo) [reply scripts/copy-iso:31]: thanks Derek, I'll update this in the next days

**Comment by elfosardo** [comment]
unsupported network type "OpenShiftSDN" in the ipv4 job, sigh

**Comment by elfosardo** [comment]
we need https://github.com/openshift/release/pull/56933

**Pull request merged**